external/emulation/emulation/zhao_carr.py

```diff
@@ -90,10 +95,36 @@ def mask_where_fortran_cloud_identical(state, emulator):
         state[Input.cloud_water],
         emulator[GscondOutput.cloud_water],
     )
-    net_condensation = cloud_out - state[Input.cloud_water]
```

Comment:
do we have any coverage of these additions?

Reply:
No coverage unfortunately.  Know the classification works based on the testing right now, but probably regtesting is the simplest approach.